apis/delivery/deliveries.py

```diff
@@ -1,15 +1,80 @@
-from flask import request, g, current_app
+from flask import request, g, current_app, send_file
 from flask_restplus import Namespace, Resource
-from flask_jwt_extended import jwt_required, decode_token
+from flask_jwt_extended import jwt_required, decode_token, get_jwt_identity
 from core import constants as C
+from core.protocol import get_delivery, listing_accessed, delivered
+from .parsers import delivery_parser
 
 api = Namespace('Delivery', description='Delivery endpoint for requesting purchased payloads')
 
-@api.route('/', methods=['GET','POST'])
+@api.route('/<string:delivery_hash>', methods=['GET'])
```

Comment:
have shied away from dynamic stuff like hashes and addresses in the path. so mulling over `/deliveries?hash=...` 

its why in `listings` i went with stuff like `/listings?owner=...` vs `/listings/owner`. imo it'll save us from "endpoint creep" by insisting that generated (addresses, hashes) data go in the query.

i.e `candidates/registration` has nothing generated or dynamic so is a :heavy_check_mark: 
but `candidates/<hash>` would be a :x:

Reply:
the only flip-side i see is that we are publishing the url and `deliveries/0xf00` looks better than `deliveries/?hash=0xf00`.

but i think that's debatable, and not sure i give af lol.

Reply:
refactored to parameter vs in the url